test_image_generation.py

```diff
@@ -0,0 +1,129 @@
+from typing import Dict, Optional
```

Comment:
This file contains lots of information:

1. some are utility functions (which can be included in the `img_utils.py` or `generate_images.py`)
2. some are for demonstration purposes (which should be in the dalle notebook). Feel free to change the dalle notebook, and I will make comments directly there. 
3. we will also need a test for the "test/agentchat/contrib/capabilities" folder. I can create that if you need help.